docs/NearFromAurora.md

```diff
@@ -0,0 +1,141 @@
+# Calling Near from Aurora, a.k.a Cross-Contract Calls (XCC)
```

Comment:
The explanation is clear but the general process is complex. Maybe adding a diagram of the flow can help them significantly.